src/MCPClient/lib/clientScripts/parse_external_mets.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python2
+
+from __future__ import print_function
+import argparse
+from lxml import etree
+import os
+import sys
+
+# archivematicaCommon
+from custom_handlers import get_script_logger
+
+from archivematicaAssignFileUUID import find_mets_file
+import parse_mets_to_db
+
+logger = get_script_logger('archivematica.mcp.client.parse_external_mets')
+
+def parse_reingest_mets(transfer_uuid, transfer_path):
+    # Parse METS to extract information needed by later microservices
+    mets_path = find_mets_file(transfer_path)
+    try:
+        root = etree.parse(mets_path)
+    except Exception:
+        print('Error parsing reingest METS', mets_path, ' - skipping')
+        logger.info('Error parsing reingest mets %s - skipping', mets_path, exc_info=True)
+        return
```

Comment:
Ideally, yes, but since stdout is displayed in the GUI and logging only goes to the log file, I put some errors in both places until that's resolved.